integration-libs/s4-service/order/components/cancel-service-order-headline/cancel-service-order-headline.component.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Component, inject } from '@angular/core';
+import { CartOutlets } from '@spartacus/cart/base/root';
+import { OrderDetailsService } from '@spartacus/order/components';
+import { Observable, map } from 'rxjs';
+
+@Component({
+  selector: 'cx-cancel-service-order-headline',
+  templateUrl: './cancel-service-order-headline.component.html',
+})
```

Comment:
```suggestion
@Component({
  selector: 'cx-cancel-service-order-headline',
  templateUrl: './cancel-service-order-headline.component.html',
  changeDetection: ChangeDetectionStrategy.OnPush,
})
```
`major:` let's keep the convention and add change detection strategy OnPush to prevent possible redundant re-rendering of this component and its child.